src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1 +1,3 @@
-UPDATE tab_container_names SET icon_name='Endosome' WHERE container_type='COMPARTMENT' AND container_name='Endosome';
+UPDATE tab_calculation_method_parameter_rates SET group_name='PERMEABILITY_CALIBRATION', read_only=0, visible=1 WHERE parameter_id=228;
+UPDATE tab_calculation_method_parameter_rates SET group_name='PERMEABILITY_CALIBRATION', read_only=0, visible=1 WHERE parameter_id=230;
+INSERT INTO tab_groups(group_name,display_name,description,parent_group,icon_name,visible,sequence,is_advanced,unique_id,pop_display_name,full_name) VALUES('PERMEABILITY_CALIBRATION','Intestinal permeability calibration','Intestinal permeability calibration','PERMEABILITY',NULL,1,6,1,152,NULL,'Intestinal permeability calibration');

```

Comment:
A new group. `Intestinal permeability calibration". Let's discuss quickly if we are happy with the name or not

Reply:
I just created a (empty) Wiki page, where we can add the detailed description how to use this calibration:
https://github.com/Open-Systems-Pharmacology/PK-Sim/wiki/Intestinal-Permeability-calibration
Can you add this link to the Description of the new group (will be  shown as a tooltip then)?

Reply:
Regarding the name: I personally is happy with it, but maybe @AndreDlm can comment on this

Reply:
done. Andrea likes the name also